### DISCUSSION TITLE
Smart Profiles stop working after a plugin sends "switchToProfile" event

### DISCUSSION THREAD

**Discussion by Miracle4142** [elgatosf/elgato#5] [General]
Hey everyone,

I just wanted to share my excitement about the move of the StreamDeckSDK subreddit to Github Discussions! I hope this will make it easier for Elgato staff to keep track of bug reports and feature requests, and it'll be awesome to see more timely responses from the team.

Speaking of bug reports, I'm a little bummed that my [previous post](https://www.reddit.com/r/StreamDeckSDK/comments/11iwuce/bug_in_smart_profiles_feature_plugin_sending/) about a bug in the Smart Profiles didn't get any attention. It's a pretty significant bug that can cause unexpected behavior for anyone relying on the "switchToProfile" API method.

To help address this, I put together a repository on GitHub that demonstrates the bug and provides step-by-step instructions on how to reproduce it. You can check it out here: https://github.com/Miracle4142/streamdeck-bug-report-switch-to-profile-event

I'm hoping that with the move to Github Discussions, we can have more productive and collaborative discussions about issues like this. I look forward to hearing your thoughts on this matter. Thank you.

**Comment by ChekTek** [comment]
Thank you for the thorough post! I apologize your Reddit post slipped through the cracks. I will review your sample plugin with the team.

**Comment by ChekTek** [comment]
This bug has been fixed and should be available with the next public release of Stream Deck! 🎉

**Comment by Miracle4142** [comment]
That's great news! I really appreciate you taking the time to address the issue, and I can't wait to see the fix in action in the next release. You guys rock, keep up the fantastic work!
